LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2015-2016 Vincent Hanquez <vincent@snarc.org>
+Copyright (c) 2015-2016 Vincent Hanquez <vincent@snarc.org> 2016-2017
```

Comment:
I think the complaint was that 2017 wasn't in the copyright year, so making it 2015-2017 would fix neil check anyway. However, if you think dropping the year is preferable and not required, do you have a link that says/explains that? Given the `neil check` tool is fairly strict and updating stuff once a year is a pain I'd be happy to get rid of the date entirely.